app/views/shared/_environment_banner.html.erb

```diff
@@ -1 +1,3 @@
-<%= environment_banner %>
+<% unless environment_banner.nil? %>
```

Comment:
I'm wondering whether because we are calling `environment_banner` twice, this might be better in the helper?

Something like:
```ruby
  def environment_banner
    return if ENVIRONMENT_COLOURS.keys.none?(sentry_env)

    banner = govuk_tag(
      text: "#{sentry_env.tr("_", " ").upcase} ENVIRONMENT",
      colour: ENVIRONMENT_COLOURS[sentry_env],
      classes: "environment-banner"
    )
    
    tag.div(banner, id: "environment-banner" aria: { label: "environment banner" }, role: "complementary)
  end

```

But, after typing that out, maybe thats not so nice either

Reply:
For me, this repetition feels okay. The original implementation feels clearer to me